legal-api/src/legal_api/resources/v2/admin.py

```diff
@@ -0,0 +1,71 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for managing an Administrative resource."""
+import uuid
+from http import HTTPStatus
+
+from flask import Blueprint, current_app
+from flask_cors import cross_origin
+from sentry_sdk import capture_message
+
+from legal_api.models import Business, UserRoles
+from legal_api.services import queue
+from legal_api.utils.auth import jwt
+from legal_api.utils.datetime import datetime
+
+
+bp = Blueprint('ADMINISTRATIVE', __name__, url_prefix='/api/v2/admin')
```

Comment:
as this is only for BN management, could it be stamped with just that instead of the whole admin namespace?